typescript/sdk/src/providers/transactions/transformer/ethersV5/schemas.ts

```diff
@@ -0,0 +1,8 @@
+import { GetCallRemoteSettingsSchema } from '../../../../middleware/account/schemas.js';
+
+export const EV5InterchainAccountTxTransformerPropsSchema =
+  GetCallRemoteSettingsSchema.pick({
+    chain: true,
+    config: true,
+    hookMetadata: true,
```

Comment:
any intended use for hookMetadata atm?

Reply:
None, just conforming to getCallRemote since that's out of scope for this change